core/point/point_test.go

```diff
@@ -13,3 +13,114 @@ func TestCreate(t *testing.T) {
 		t.Errorf("p.Y()=%v, expected %v", gotY, 2)
 	}
 }
+
+func TestPointToSGFTranslate(t *testing.T) {
+	// First test translation from integer-point to SGF-string-point
+	testToSGFCases := []struct {
+		desc string
+		in   *Point
+		want string
+	}{
+		{
+			desc: "Point => SGF",
+			in: &Point{
+				x: 8,
+				y: 16,
+			},
+			want: "iq",
+		},
+		{
+			desc: "Point => SGF",
+			in: &Point{
+				x: 12,
+				y: 5,
+			},
+			want: "mf",
+		},
+		{
+			desc: "Point => SGF",
+			in: &Point{
+				x: 33,
+				y: 8,
+			},
+			want: "Hi",
+		},
+		{
+			desc: "Point => SGF",
+			in: &Point{
+				x: 40,
+				y: 51,
+			},
+			want: "OZ",
+		},
+	}
+
+	for _, tc := range testToSGFCases {
+		t.Run(tc.desc, func(t *testing.T) {
+			toSGFOut, _ := New(tc.in.x, tc.in.y).ToSGF()
+			if toSGFOut != tc.want {
+				t.Errorf("%q.ToSGF() = %q, but wanted %q", tc.in,
+					toSGFOut, tc.want)
+			}
+		})
+	}
+}
+
+func TestSGFToPointTranslate(t *testing.T) {
+	testToPointCases := []struct {
+		desc string
+		in   string
+		want *Point
+	}{
+		{
+			desc: "SGF => Point",
+			in:   "iq",
+			want: &Point{
+				x: 8,
+				y: 16,
+			},
+		},
+		{
+			desc: "SGF => Point",
+			in:   "mf",
+			want: &Point{
+				x: 12,
+				y: 5,
+			},
+		},
+		{
+			desc: "SGF => Point",
+			in:   "Hi",
+			want: &Point{
+				x: 33,
+				y: 8,
+			},
+		},
+		{
+			desc: "SGF => Point",
+			in:   "OZ",
+			want: &Point{
+				x: 40,
+				y: 51,
+			},
+		},
+	}
+
+	for _, tc := range testToPointCases {
+		t.Run(tc.desc, func(t *testing.T) {
+			toPointOut, _ := NewFromSGF(tc.in)
+			// Utilizing the point.go *Point type X Y getters below
+			pointX := toPointOut.X()
```

Comment:
you can just have x := toPointOut.X(). Or, honestly, you probably don't need this variable. blah.X() is pretty short.